docs/Contributing.asciidoc

```diff
@@ -431,10 +499,246 @@ Also find more details in
+
+* `OPENQA_BASEDIR=$HOME/openqadev.leap`
+* Database name: `openqa-local`
+* Database user: `geekotest`
+* Database container name: `postgres-openqa`
+
+==== Environment Variables
+
+Create a file that you can `source` whenever you want to work in this
+environment:
+
+File: `~/localopenqa.leap.sh`:
+[source,sh]
+----
+#!/bin/bash
+
+# The directory where openQA operates in and creates everything in:
+# * git repos
+# * openQA assets, pool directory, testresults
+# * configuration
+
+export OPENQA_BASEDIR=$HOME/openqadev.leap
+
+# The rest of the variables can likely stay like this
+
+# If you want to import database dumps from elsewhere
+#export SQL_DUMP_DIR=$HOME/sqldumps
+
+export OPENQA_CONFIG=$OPENQA_BASEDIR/config
+export OPENQA_REPOS="$OPENQA_BASEDIR/git"
+PATH=$OPENQA_REPOS/openQA/scripts:$PATH
+
+# ----------- CPAN ------------
+# Whenever necessary, you can install additional modules from CPAN like:
+# cpanm -l $OPENQA_BASEDIR/perl5 Some::Module@1.23
+export PERL5LIB=$OPENQA_BASEDIR/perl5/lib/perl5
+PATH=$OPENQA_BASEDIR/perl5/bin:$PATH
+
+export OPENQA_KEY=1234567890ABCDEF
+export OPENQA_SECRET=1234567890ABCDEF
+----
+
+==== Postgres Container
+
+To be able to keep the database when stopping the container, we create a
+named volume.
+We are using a Debian postgres container here, but it doesn't matter what
+you use.
+[source,sh]
+----
+### host ###
+podman volume create mypostgres
+podman run -it --rm --name postgres-openqa \
+    -v mypostgres:/var/lib/postgresql/data \
+    -p 5432:5432 \
+    -e POSTGRES_PASSWORD="x" \
+    -d \
+    docker.io/library/postgres:17
+
+# Optionally add this to mount a directory for sql dumps to import
+#    -v "$SQL_DUMP_DIR:/sqldumps" \
+----
+
+For the next step you need to execute commands in the container:
+
+[source,sh]
+----
+# Create the role and database
+podman exec -it postgres-openqa su - postgres
+
+### postgres container ###
+createuser --no-createdb --pwprompt geekotest
+# enter password ('x' in this example)
+createdb -O geekotest openqa-local
+----
+
+==== Distrobox
+
+You can use a Leap container and install the requirements, but you can also
+build your own container with the requirements already installed.
+
+[source,sh]
+----
+### host ###
+distrobox create -i registry.opensuse.org/opensuse/leap:15.6 -n box-openqadev-leap
+distrobox enter box-openqadev-leap
+# Tumbleweed:
+#distrobox create -i registry.opensuse.org/opensuse/tumbleweed:latest -n box-openqadev-tw
+
+### distrobox container ###
+
+# Leap 15.6
+zypper addrepo -G -p 95 https://download.opensuse.org/repositories/devel:/openQA/15.6 devel-openqa \
+    && zypper addrepo -G -p 95 https://download.opensuse.org/repositories/devel:/openQA:/Leap:/15.6/15.6/ devel-openqa-leap
+
+# Tumbleweed
+#zypper addrepo -G p 95 https://download.opensuse.org/repositories/devel:/openQA/openSUSE_Tumbleweed devel-openqa \
+#        && zypper addrepo -G -p 95 https://download.opensuse.org/repositories/SUSE:/CA/openSUSE_Tumbleweed/SUSE:CA.repo
```

Comment:
Instead of repeating how about pointing to https://github.com/os-autoinst/openQA/blob/master/docs/Installing.asciidoc#development-version-repository instead. This way we also don't need to change the version reference every year

Reply:
I think I was asked to add Tumbleweed here in a previous review.
I can remove it again...

Reply:
Leave it if it helps getting this PR merged 😉